sap-s4hana-cloud/2.2/modules/ROOT/pages/sap-s4hana-cloud-connector-studio.adoc

```diff
@@ -0,0 +1,151 @@
+= Use Studio to Configure SAP S/4HANA OData Connector 2.2 - Mule 4
+:page-aliases: connectors::sap/sap-s4hana-cloud-connector-studio.adoc
+
+Anypoint Studio (Studio) editors help you design and update your Mule applications, properties, and configuration files.
+
+To add and configure a connector in Studio:
+
+. <<create-mule-project,Create a Mule project>>.
+. <<add-connector-to-project,Add the connector to your Mule project>>.
+. <<configure-input-source,Configure an input source for the connector's flow>>.
+. <<add-connector-operation,Add a connector operation to the flow>>.
+. <<configure-global-element,Configure a global element for the connector>>.
+. <<configure-other-fields,Configure the other connector fields>>.
+
+
+If you are new to configuring connectors in Studio, see xref:connectors::introduction/intro-config-use-studio.adoc[Use Studio to Configure a Connector]. If, after reading this topic, you need additional information about the connector fields, see the <link to the connector Reference Guide [<connector name> Reference].
+
+[[create-mule-project]]
+== Create a Mule Project
+
+In Anypoint Studio, create a new Mule project in which to add and configure the connector:
+
+. From the Studio main menu, select *File > New > Mule Project*.
+. Enter a name for your Mule project and click *Finish*.
+
+[[add-connector-to-project]]
+== Add the Connector to Your Mule Project
+
+Add Anypoint Connector for SAP S/4HANA OData to a Mule project to automatically populate the project's XML code with the connector's namespace and schema location, as well as to add the required dependencies to the project's `pom.xml` file.
+
+. In the *Mule Palette* view, click *(X) Search in Exchange*.
+. In *Add Dependencies to Project*, type `SAP S/4HANA OData` in the search field.
```

Comment:
We are using lowercase now for the search term to make it easier for the user's to type.